app/workers/email/send_user_digest_worker.rb

```diff
@@ -0,0 +1,23 @@
+module Email
+  class SendUserDigestWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :low_priority, retry: 15, lock: :until_executing
```

Comment:
There will be 300k of these enqueued at once, I wanted them on the low priority queue to ensure they didn't block anything else that was more time sensitive